src/app.js

```diff
@@ -1,20 +1,64 @@
-const express = require('express');
-const app = express();
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const morgan = require('morgan')
+const bodyParser = require('body-parser')
 const cors = require('cors')
-const favicon = require('express-favicon');
-const logger = require('morgan');
+const favicon = require('express-favicon')
+const mainRouter = require('./routes/mainRouter.js')
+const helmet = require('helmet')
+const session = require('express-session')
+const passport = require('passport')
+const mongoose = require('mongoose')
+const asyncErrors = require('express-async-errors')
```

Comment:
The docs for this library indicate that only need to require it, in order for it to work (https://www.npmjs.com/package/express-async-errors)

You can see in the Github code for that library, that it doesn't export anything: https://github.com/davidbanham/express-async-errors/blob/master/index.js

If you were to do a `console.log(asyncErrors)` you'll see that it just is logging an empty object `{}`.

So all we need on this line is:
```suggestion
require('express-async-errors')
```

and we should remove line 42-43.

This is similar to this exercise in the Node course, where we call `require('./07-mind-grenade')` to run the code in that file without saving anything to a variable: https://github.com/Code-the-Dream-School/node-express-course/blob/main/01-node-tutorial/03-modules.js

Reply:
@DYA13  Not sure if you saw this comment since I think I reviewed it on an older commit

Reply:
Sorry Akos, I didn't see it , now I have changed commit- d777f3cb45232dcd987b186f5c6544f67714967f